physical/raft/raft.go

```diff
@@ -1709,21 +1649,41 @@ func (b *RaftBackend) applyLog(ctx context.Context, command *LogData) error {
 		return errors.New("entries on FSM response were empty")
 	}
 
-	for i, logOp := range command.Operations {
-		if logOp.OpType == getOp {
-			fsmEntry := fsmar.EntrySlice[i]
+	if len(command.Operations) < 1 {
```

Comment:
We should expand the tests to handle `raftchunking.ChunkingApply` + transaction interaction.

Reply:
This works fine, but we'll probably want to run it separately as it blows up the tests a lot to run it frequently.